src/main/java/org/demo/conf/cxbox/extension/notification/NotificationServiceImpl.java

```diff
@@ -69,7 +75,30 @@ public void delete(List<Long> ids) {
 
 	@Override
 	public Long getCountWithMark(Boolean mark) {
-		return notificationRepository.countByUserAndIsRead(userRepository.getReferenceById(sessionService.getSessionUser().getId()), mark);
+		return notificationRepository.countByUserAndIsRead(userRepository.getReferenceById(sessionService.getSessionUser()
+				.getId()), mark);
+	}
+
+
+	private NotificationDTO notificationEntityToDto(Notification notification) {
```

Comment:
map entity to dto in dto constructor or in mapper, but not both ways in one feature service